File: cs/Tapstream/CoreListenerImpl.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

#if TEST_WINPHONE || WINDOWS_PHONE
#else
using System.Threading.Tasks;
#endif

namespace TapstreamMetrics.Sdk
{
    class CoreListenerImpl : CoreListener
    {
        public void ReportOperation(string op)
        {
        }

        public void ReportOperation(string op, string arg)
        {
        }
    }
}

File: java/TapstreamTest/src/main/java/com/tapstream/sdk/CoreListenerImpl.java
package com.tapstream.sdk;

class CoreListenerImpl implements CoreListener {
	public OperationQueue queue;

	public CoreListenerImpl(OperationQueue queue) {
		this.queue = queue;
	}

	public void reportOperation(String op) {
		queue.add(new Operation(op, null));
	}

	public void reportOperation(String op, String arg) {
		queue.add(new Operation(op, arg));
	}
}